css-layout-classes/styles.css

```diff
@@ -0,0 +1,30 @@
+#container {
+  max-width: 800px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+  flex-direction: row;
```

Comment:
you don't need flex direction or justify content in this selector, just display flex

Reply:
<img width="344" alt="image" src="https://user-images.githubusercontent.com/113648549/195716821-d573e900-d7d5-4c41-bd78-243a951ccd8e.png">